index.js

```diff
@@ -74,8 +108,34 @@ function makeGrid(){
     }
     gridWrapper.appendChild(fragment);
     Wrapper.appendChild(gridWrapper);
+    borderGrid();
+    setGridDivWidth();
 };
 
+function setGridDivWidth(){
+    var div = document.querySelector("#Wrapper > div");
+    div.style.width = (gridEdgeLenght + "px");
+}
+
+function borderGrid(){
+    var borderTop = document.querySelectorAll(".row-1 .cell");
+    var borderRight = document.querySelectorAll(".cell-" + gridEdge);
+    var borderBottom = document.querySelectorAll(".row-" + gridEdge + " .cell");
+    var borderLeft = document.querySelectorAll(".cell-1");
+    for(i = 0; i <= borderTop.length - 1; i++){
```

Comment:
You could add class `border-edge` to those elements. And add appropriate styles to CSS.